cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/controller/BoardsAndWidgetsController.kt

```diff
@@ -11,39 +10,64 @@ import io.vertx.core.AbstractVerticle
 import io.vertx.core.json.JsonObject
 import io.vertx.core.logging.Logger
 import io.vertx.core.logging.LoggerFactory
+import com.cognifide.cogboard.CogboardConstants as CC
 
-internal open class BoardsController : AbstractVerticle() {
+class BoardsAndWidgetsController : AbstractVerticle() {
```

Comment:
I had to merge `BoardsController` and `WidgetsController` back together because they have to share the same `widgets` map object.